docs/setup/clouds/aws.md

```diff
@@ -0,0 +1,59 @@
+# Google Cloud
+
+The runner have the capability to create resources on AWS. Follow the steps below to authenticate
+and configure this service
+
+## Credentials
+
+### Get credentials for Engineering-EC2User
+
+Go on AWS Single Sign-on to get the credentials.
+Then click on the link called "Command Line or programmatic access".
+Get :
+- AWS Access Key Id
+- AWS Secret Access Key
+- AWS Session Token
+
+Create a .env file at the root of the project and put in it :
+AWS_ACCESS_KEY_ID="XXX"
+AWS_SECRET_ACCESS_KEY="XXX"
+AWS_SESSION_TOKEN="XXX"
```

Comment:
Remember some information here only apply to us, we don't need to document that on a project that is public.

Mentioning that you need an access key id and a secret access key should be enough.

Reply:
Yes indeed, I correct everything